packages/components-js/src/pages/carousel.html

```diff
@@ -252,3 +252,27 @@ <h2 slot="heading">
     <div>Slide 3</div>
   </p-carousel>
 </div>
+
+<div class="playground light" title="should render carousel with alignHeader=center on light background">
+  <p-carousel heading="Aligned header" align-header="center" description="Description">
+    <div>Slide 1</div>
+    <div>Slide 2</div>
+    <div>Slide 3</div>
+  </p-carousel>
+</div>
+
+<div
+  class="playground light"
+  title="should render carousel with alignHeader=center and wrap-content=true on light background"
```

Comment:
Here we got a mix of `camelCase` and `param-case`. I'd suggest to stick to `param-case` since this is the format for attributes.

Reply:
Fixed